example/.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This file doesn't match what I see in the [Drupal Code Repo](http://cgit.drupalcode.org/drupal/tree/.eslintrc), though maybe the current PR description means for it to be updated. My two questions are:

1. Does the D8 JS coding standard match what is expected of D7?
2. If (1) leads us to separate configs, I think we *should not* maintain our own copy of the D8 config. Instead we should do something clever like copy it from `build/html`.